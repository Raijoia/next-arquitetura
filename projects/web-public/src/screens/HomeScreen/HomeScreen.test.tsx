import { HomeScreen } from "./HomeScreen";
import { customRender } from "@rai/test-commons/react-testing-library";

function CustomWrapper({ children }) {
  return <div>{children}</div>;
}

const render = customRender(CustomWrapper);

describe("<HomeScreen />", () => {
  it("renders the page", () => {
    const { container } = render(<HomeScreen />);

    expect(container).toMatchSnapshot();
  });
});
