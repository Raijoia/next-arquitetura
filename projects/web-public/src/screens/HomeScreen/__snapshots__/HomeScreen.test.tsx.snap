// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeScreen /> renders the page 1`] = `
<div>
  <div>
    <main>
      <h1
        class="glaJeq"
      >
        Home
      </h1>
      <p
        class="glaJeq"
      >
        Importando modulo local: @rai/utils/math/sum sum(2, 2): 
        4
         
      </p>
    </main>
  </div>
</div>
`;
